src/Microsoft.TemplateEngine.Edge/Template/WellKnownSearchFilters.cs

```diff
@@ -181,5 +181,31 @@ public static class WellKnownSearchFilters
                 return null;
             };
         }
+
+        public static Func<ITemplateInfo, MatchInfo?> AuthorFilter(string author)
```

Comment:
minor: would a normal method suffice instead?

Reply:
what do you mean by "normal"?
- non-static won't work
- internal - probably, but it would be nice to expose it as for other filters. VS uses these methods.